packages/utils/regl-renderer/src/geometry-utils/entitiesFromSolids.js

```diff
@@ -56,7 +56,7 @@ const entitiesFromSolids = (params, solids) => {
     const visuals = {
       drawCmd: 'drawMesh',
       show: true,
-      color: [0.8, 0.5, 0.7, 0.1],
+      color: meshColor,//[0.8, 0.5, 0.7, 1.0],
       transparent: geometry.isTransparent, // not sure
```

Comment:
I think this is the correct change. Why leave the comment with the old color?

Reply:
agreed